test/test_docker_utils.py

```diff
@@ -1,4 +1,5 @@
 import os  # NOQA
+import unittest
```

Comment:
can u switch to pytest? we don't use unittest (if we do, they should be converted to pytest) as pytest is much better.